src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configurations/TargetFrameworkProjectConfigurationDimensionProvider.cs

```diff
@@ -59,12 +59,6 @@ public TargetFrameworkProjectConfigurationDimensionProvider(IProjectLockService
                 var configuredProject = await project.GetSuggestedConfiguredProjectAsync().ConfigureAwait(false);
                 var msbuildProject = await access.GetProjectAsync(configuredProject).ConfigureAwait(false);
                 
-                // If the project already defines a specific "TargetFramework" to target, then this is not a cross-targeting project and we don't need a target framework dimension.
-                if (msbuildProject.Properties.Any(p => p.Name.Equals(TargetFrameworkPropertyName, StringComparison.OrdinalIgnoreCase) && !string.IsNullOrEmpty(p.EvaluatedValue)))
```

Comment:
This was an incorrect check that I had added in my previous checkin - we should consider a project to be always a cross-targeting project as long as it has "TargetFrameworks" property set.